packages/cashier/src/stores/account-transfer-store.ts

```diff
@@ -377,7 +377,7 @@ export default class AccountTransferStore {
             const cfd_platforms = {
                 mt5: { name: 'Deriv MT5', icon: 'IcMt5' },
                 dxtrade: { name: 'Deriv X', icon: 'IcRebranding' },
-                derivez: { name: 'Deriv EZ', icon: 'IcDerivez' },
+                derivez: { name: 'Deriv EZ', icon: 'IcRebranding' },
```

Comment:
```suggestion
                derivez: { name: 'Deriv EZ', icon: 'IcRebrandingDerivEz' },
```
👀 ?

Reply:
Hi @yashim-deriv 🙂 ,
`DerivEZ` appended to the icon name is provided by the `getCFDAccount()` which is imported for cfd.ts.